docs/AboutUs.md

```diff
@@ -9,15 +9,14 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 ## Project team
 
-### John Doe
+### Ng Wee Han
```

Comment:
No changes required here, Boon hai will update all the AboutUs information on his side.

Reply:
I have reverted the changes made on my side